docs/diagrams/SortItemsSequenceDiagram.puml

```diff
@@ -0,0 +1,91 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":FoodRemParser" as FoodRemParser LOGIC_COLOR
+participant ":SortCommandParser" as SortCommandParser LOGIC_COLOR
+participant "s:SortCommand" as SortCommand LOGIC_COLOR
+participant ":ItemNameComparator" as ItemNameComparator LOGIC_COLOR
+participant "c:ChainComparator" as ChainComparator LOGIC_COLOR
+participant ":CommandResult" as CommandResult LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+end box
+
+[-> LogicManager : execute("sort n/")
+activate LogicManager
+
+LogicManager -> FoodRemParser : parseCommand("sort n/")
+activate FoodRemParser
+
+create SortCommandParser
+FoodRemParser -> SortCommandParser
+activate SortCommandParser
+
+SortCommandParser --> FoodRemParser
+deactivate SortCommandParser
+
+FoodRemParser -> SortCommandParser : parse("/n")
+activate SortCommandParser
+
+create ItemNameComparator
+SortCommandParser -> ItemNameComparator
+activate ItemNameComparator
+
+ItemNameComparator --> SortCommandParser
+deactivate ItemNameComparator
+
+create ChainComparator
+SortCommandParser -> ChainComparator
+activate ChainComparator
+
+ChainComparator --> SortCommandParser
+deactivate ChainComparator
+
+create SortCommand
+SortCommandParser -> SortCommand
+activate SortCommand
+
+SortCommand --> SortCommandParser :
+deactivate SortCommand
+
+SortCommandParser --> FoodRemParser :
+deactivate SortCommandParser
+'Hidden arrow to position the destroy marker below the end of the activation bar.
+SortCommandParser -[hidden]-> FoodRemParser
+destroy SortCommandParser
+
+FoodRemParser --> LogicManager :
+deactivate FoodRemParser
+
+LogicManager -> SortCommand : execute()
```

Comment:
Should we include model as a parameter then?
If it's not important, you can ignore this, then we can just standardize by not having model as a parameter in all our execute method.

Reply:
I'll leave it out for now, if we need to add it in we can always have one PR that adds it to all our UMLs.

Reply:
Actually, scratch that: I'll add it in for this UML diagram and the Tagging UML diagram #230